### DISCUSSION TITLE
QERaman

### DISCUSSION THREAD

**Discussion by Santoshkumarkaushik** [nguyen-group/QERaman#6] [Q&A] [answered]
We are getting four files as output for QERaman for four elaser 1.95 eV, 2.33 eV, 2.54 eV , 2.78 eV.Which file we have to use for comparison with experimental raman spectra. ?

**Comment by nguyen-group** [comment, marked as answer]
It depends on the Raman laser of the experiment. For example, 2.33 eV is 532 nm.
